src/Detector/Hugo/HugoConstants.cs

```diff
@@ -0,0 +1,18 @@
+// This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+namespace Microsoft.Oryx.Detector.Hugo
+{
+    public static class HugoConstants
+    {
+        public const string Version = "0.71.0";
+        public const string PlatformName = "hugo";
+        public const string InstalledHugoVersionsDir = "/opt/hugo";
+        public const string InstallationUrlFormat = "https://github.com/gohugoio/hugo/releases/download/v#VERSION#/#TAR_FILE#";
```

Comment:
we can exclude some constants that are not used

Reply:
I want to handle this in a separate PR. I created a new PR to update our tool which generates constants to look for multiple constants files. Once that is in, I want to create a separate Detector specific constants file.